docker/docker-compose.yml

```diff
@@ -1,81 +1,88 @@
 version: '3.6'
 services:
-    kong:
-      container_name: supabase-kong
-      build:
-          context: ./kong
-      environment:
-          KONG_DECLARATIVE_CONFIG: /var/lib/kong/kong.yml
-          KONG_PLUGINS: request-transformer,cors,key-auth,http-log
-      ports:
+  kong:
+    container_name: supabase-kong
+    build:
+      context: ./kong
+    environment:
+      KONG_DECLARATIVE_CONFIG: /var/lib/kong/kong.yml
+      KONG_PLUGINS: request-transformer,cors,key-auth,http-log
+    ports:
       - 8000:8000/tcp
       - 8443:8443/tcp
-    auth:
-      container_name: supabase-auth
-      image: supabase/gotrue:latest
-      ports:
-        - '9999:9999'
-      environment:
-        GOTRUE_JWT_SECRET: super-secret-jwt-token-with-at-least-32-characters-long
-        GOTRUE_JWT_EXP: 3600
-        GOTRUE_JWT_DEFAULT_GROUP_NAME: authenticated
-        GOTRUE_DB_DRIVER: postgres
-        DB_NAMESPACE: auth
-        API_EXTERNAL_URL: localhost
-        GOTRUE_API_HOST: 0.0.0.0
-        PORT: 9999
-        GOTRUE_DISABLE_SIGNUP: "false"
-        GOTRUE_SITE_URL: localhost
-        GOTRUE_MAILER_AUTOCONFIRM: "true"
-        GOTRUE_LOG_LEVEL: DEBUG
-        GOTRUE_OPERATOR_TOKEN: super-secret-operator-token
-        DATABASE_URL: "postgres://postgres:postgres@db:5432/postgres?sslmode=disable"
-      depends_on:
-        - db
-    rest:
-      container_name: supabase-rest
-      image: postgrest/postgrest:latest
-      ports:
-        - '3000:3000'
-      depends_on:
+  auth:
+    container_name: supabase-auth
+    image: supabase/gotrue:latest
+    ports:
+      - $AUTH_PORT
```

Comment:
looks like these need brackets right?:

https://docs.docker.com/compose/environment-variables/#substitute-environment-variables-in-compose-files

eg: `${AUTH_PORT}`

Reply:
both is possible. i couldn't find documentation on what would be the advantage of using or not using brackets, besides my personal preference of not using them becaus of simplicity/looks reasons.

Reply:
> both options are possible

I couldn't find any docs showing it without brackets - I'm just worried it's not supported by some platforms/tools?

Otherwise this PR is great @firstdorsal. If we can change the vars to `${}` (or you can point me to some docs without brackets) then I'm excited to merge this - huge improvement for DX

Reply:
no problem :) i added the brackets.
i'm also working on a traefik compose.

Reply:
OK this looks great now 🚀 

> I'm also working on a traefik compose.

I noticed the new file in the PR. WIll you also add docs for this one? I'm not familiar with traefik, so I won't be able to document the changes myself

Reply:
i provided the readme file for the directory.
i also can provide further documentation where needed.